### ISSUE TITLE
Add `TagHelperOutput` extension methods for `class` value manipulation

### ISSUE THREAD

**Issue by dougbu** [aspnet/Mvc#6817]
Many thanks to @Elfocrash for his work in #6565. That PR adds the following extension methods:
``` c#
public static void AddClass(
    this TagHelperOutput tagHelperOutput,
    string classValue,
    HtmlEncoder htmlEncoder) { }
public static void RemoveClass(
    this TagHelperOutput tagHelperOutput,
    string classValue,
    HtmlEncoder htmlEncoder) { }
```

----

Issue created in arrears to track work done in the 2.1.0 milestone.

**Comment by dougbu** [comment]
db397d81

**Comment by kshyju** [comment]
Thanks! 

Is this released ? Can't find this extension method  (.NET core 2.0 project)

**Comment by Elfocrash** [comment]
@kshyju This was merged after the 2.0 release. You can check on the right-hand side of the page that the Milestone for this is 2.1.0-preview1 so it will be out with 2.1.0

**Comment by kshyju** [comment]
Perfect! Thanks. Did not notice the milestone label ;)
